libs/vm/vm.py

```diff
@@ -1,5 +1,3 @@
-from __future__ import annotations
```

Comment:
you still need this.

Reply:
No need if minimum python is 3.12

Reply:
I thought they are supposed to become mandatory? https://docs.python.org/3/library/__future__.html#id1.

Reply:
supported from python 3.10 https://peps.python.org/pep-0604/